ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.js

```diff
@@ -0,0 +1,490 @@
+      netRegexFr: NetRegexes.startsUsing({ id: '618E', source: 'Noyau Orange', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '618E', source: '赤球', capture: false }),
+      durationSeconds: 4,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Switch to black',
+          de: 'Auf Schwarz wechseln',
+        },
+      },
+    },
+    {
+      id: 'Paradigm Meng-Zi/Xun-Zi Universal Assault',
+      netRegex: NetRegexes.startsUsing({ id: '5C06', source: ['Meng-Zi', 'Xun-Zi'], capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5C06', source: ['Meng-Zi', 'Xun-Zi'], capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5C06', source: ['Meng-Zi', 'Xun-Zi'], capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5C06', source: ['モウシ', 'ジュンシ'], capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 4,
+      suppressSeconds: 5,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Paradigm False Idol Screaming Score',
+      netRegex: NetRegexes.startsUsing({ id: '5BDD', source: 'False Idol', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5BDD', source: 'Ihre Abgöttlichkeit', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5BDD', source: 'Déesse Factice', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5BDD', source: '偽造サレタ神', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 4,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Paradigm False Idol Made Magic Left',
+      netRegex: NetRegexes.startsUsing({ id: '5BD6', source: 'False Idol', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5BD6', source: 'Ihre Abgöttlichkeit', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5BD6', source: 'Déesse Factice', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5BD6', source: '偽造サレタ神', capture: false }),
+      durationSeconds: 5,
+      response: Responses.goRight(),
+    },
+    {
+      id: 'Paradigm False Idol Made Magic Right',
+      netRegex: NetRegexes.startsUsing({ id: '5BD7', source: 'False Idol', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5BD7', source: 'Ihre Abgöttlichkeit', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5BD7', source: 'Déesse Factice', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5BD7', source: '偽造サレタ神', capture: false }),
+      durationSeconds: 5,
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'Paradigm False Idol Darker Note You',
+      netRegex: NetRegexes.headMarker({ id: '008B' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 4,
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Tank Buster and Spread',
```

Comment:
This is kinda wordy.  I think spread should come first, at least.  But I'd suggest `Spread + Tank Buster` or even just `Tank Cleave`?

This is also a tiny thing (which you should feel very free to disagree about) but I wonder about putting this output string in the `Paradigm False Idol Darker Note` trigger, so then you only have one collect trigger (no output) and one output trigger (that handles all players).

Reply:
I mean, the root of the callout here is: You have a spread circle that does tank buster damage, and if the circles overlap you will kill both of you instantly.
It might be just fine to say "Tank Buster" if the spread + no overlap is obvious

Reply:
Yeah, I guess I'm asking how it's different than the previous tankbusters in the alliance raid where all three tanks get marked with a (presumed) tank buster cleave and also need to spread there.  This one has a marker and a cute circle, but it feels somewhat like the mechanic is the same in both cases, is my line of thinking.

Mostly my worry about the "and spread" part is that there's no avoiding the buster, it's a "spread so you can take this tank buster" not "take this tank buster...and then book it somewhere else", which is what the text might make somebody think.